### PR TITLE
Add tests for form submission on login/logout and CreateAccount.

### PR DESCRIPTION
**Pull request by the-oem**
## What's this PR do?
During my project eval, I was seeking help on testing function calls on form submission. Taylor explained how to do it and wanted me to get those tests added.

## Pre-Merge TODOs
 - [ ] Tests Added
   - Login.test.js
   - CreateAccount.test.js

## Learning

Learned how to mock out preventDefault as part of testing form submission.

## Where should the reviewer start?

No concerns. Just start with looking at the tests that changed. There is one new complete test in both files.

## Any background context you want to provide?
N/A

## Screenshots (if appropriate)
N/A

## What gif best describes this PR or how it makes you feel?
image

## Reviewers & client
@Ggoering
@jsullivan5
@Tman22 (client)

### CODE REVIEW

File: app/components/Login/Login.test.js [hunk 1]
```diff
@@ -51,12 +57,14 @@ describe('Login Component Testing', () => {
...
     const loginButton = wrapper.find('.btn-login');
     const loginForm = wrapper.find('.login-form');
 
-    loginForm.simulate('submit');
+    loginForm.simulate('submit', { preventDefault: () => {} });
+
+    expect(handleAuthFn).toHaveBeenCalledTimes(1);
   });
 
   it('should render a welcome message if you are logged in', () => {
```
Comment (Ggoering) [inline app/components/Login/Login.test.js:69]: I'm a little confused here. Why does the it block reference a welcome message but the tests do represent it?

Comment (Ggoering) [reply app/components/Login/Login.test.js:69]: dont*

Comment (the-oem) [reply app/components/Login/Login.test.js:69]: The tests DO represent it, on line 71 of the file. That isn't what changed or have anything to do with the PR, though.

Comment (Ggoering) [reply app/components/Login/Login.test.js:69]: My bad! Didn't realize git was skipping lines in these file change views

**Review by Ggoering — COMMENTED** [review]
Looks good!

**Pull request merged**